tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -714,7 +714,6 @@ def _port_status(expected_state):
         pytest_assert(wait_until(PORT_TOGGLE_TIMEOUT, 2, 0, _port_status, 'up'),
                         "Interface {} should be admin up".format(test_intf))
 
-
```

Comment:
Harsha, seems this line is not necessary to delete.  Could we remove the change on test_ifcace_namingmode.py?